pybamm_tea/tea.py

```diff
@@ -303,14 +307,47 @@ def initialize(self):
                     * pava.get("Maximum concentration in positive electrode [mol.m-3]")
                 )
             )
-        self.parameter_values = {**self.parameter_values, **pava}
+        if (
+            pava.get("Initial concentration in negative electrode [mol.m-3]") is None
+            and pava.get("Initial concentration in positive electrode [mol.m-3]") is None
+        ):
+            # initialize concentrations based on initial loss of lithium inventory
+            pava["Initial concentration in negative electrode [mol.m-3]"] = 0
+            pava["Initial concentration in positive electrode [mol.m-3]"] = pava.get("Maximum concentration in positive electrode [mol.m-3]")
+            warnings.warn("Warning: 'Initial concentration in negative electrode [mol.m-3]' and 'Initial concentration in positive electrode [mol.m-3]' are set to 0 and maximum concentration in positive electrode, respectively.")
+        if (pava.get("Initial concentration in negative electrode [mol.m-3]") is None
+            and pava.get("Initial concentration in positive electrode [mol.m-3]") is not None
+        ):
+            raise ValueError("Error: 'Initial concentration in negative electrode [mol.m-3]' missing. Please, supply both or none.")
+        if (
+            pava.get("Initial concentration in negative electrode [mol.m-3]") is not None
+            and pava.get("Initial concentration in positive electrode [mol.m-3]") is None
+        ):
+            raise ValueError("Error: 'Initial concentration in positive electrode [mol.m-3]' missing. Please, supply both or none.")
+        if pava.get("Initial loss of lithium inventory") is None:
+            pava["Initial loss of lithium inventory"] = 0
+            warnings.warn("Warning: 'Initial loss of lithium inventory' is set to 0.")
+        if (
```

Comment:
by this point neither should be None, based on the above logic

Reply:
Thank you for noticing, I'll remove the condition.